src/coveralls.erl

```diff
@@ -338,10 +345,40 @@ replce_char_test_() ->
   , ?_assertEqual("foobarfoo", replace_char("foo\"foo", $", "bar")) %"
   ].
 
-fullpath_test_() ->
-  [ ?_assertEqual("/a/b", fullpath(["/", "a", "b"], []))
-  , ?_assertEqual("a/c", fullpath(["a", "b", "..", ".", "c"], []))
-  , ?_assertEqual("/", fullpath(["..", ".", "/"], []))
+expand_test_() ->
+  [ ?_assertEqual("/a/b", expand(["/", "a", "b"], []))
+  , ?_assertEqual("a/c", expand(["a", "b", "..", ".", "c"], []))
+  , ?_assertEqual("/", expand(["..", ".", "/"], []))
+  ].
+
+realpath_and_relative_test_() ->
```

Comment:
It would be better to use os:cmd("mktemp -d") to create a temporary directory and use that. This would also remove the need to do any cleanup.